lib/index.js

```diff
@@ -170,6 +170,12 @@ module.exports = (bookshelf) => {
             }
             operator = this.operators[operator];
             value = this.sanitizeVal(key, value, operator);
+
+            if (_.isArray(value) && value.length <= 1 && (operator === 'IN' || operator === 'NOT IN')) {
+              operator = operator === 'IN' ? '=' : '!=';
+              value = value.length ? value[0] : '____NOT_A_VALUE____';
```

Comment:
I know this arbitrary string is odd ... not sure if there's a better way? I think it works with all column types (string, number, binary, etc.)? And that way if the query is nested in an '_and' or '_or' or something then it should work properly, I believe...